nix/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import ./nixpkgs-src.nix {} }:
+
+let
+  haskell = import (./haskell.nix-src.nix) { inherit pkgs; };
+
+  pkgSet = haskell.mkStackPkgSet {
+    stack-pkgs = import ./pkgs.nix;
+    pkg-def-extras = [];
+    modules = [
+      {
+        # The SQLite tests break due to not finding 'echo'. We disable them for now.
+        packages.funflow.components.tests.unit-tests.testFlags = ["-p '! /SQLite/'"];
```

Comment:
Maybe a missing dependency on `coreutils`?

Reply:
Right, but also something about how/where we're searching for binaries.

Reply:
Specifically: simply adding `coreutils/bin` to the path didn't actually fix this problem!